src/starcheck.pl

```diff
@@ -136,6 +138,23 @@
 my $dither_file= get_file("$par{dir}/History/DITHER.txt*",'dither'); 
 my $radmon_file= get_file("$par{dir}/History/RADMON.txt*", 'radmon');
 
+# Check for characteristics.  Ignore the get_file required vs not API and just pre-check
+# to see if there is characteristics
+my $char_file;
+my $char_glob = "$par{dir}/${sosa_dir_slash}mps/ode/characteristics/CHARACTERIS*";
+my $char_check = glob($char_glob);
+if ($char_check){
```

Comment:
Should be written `if (glob($char_glob)){`.  There is no reason to create a one-off variable in this short context.